spec/presenters/tree_builder_archived_spec.rb

```diff
@@ -0,0 +1,103 @@
+describe TreeBuilderArchived do
+  context 'TreeBuilderArchived' do
+    before do
+      extend TreeBuilderArchived
+      allow(self).to receive(:count_only_or_objects_filtered) do |count_only, objects, name|
+        count_only ? objects.size : objects.sort_by{ |object| object[name.to_sym] }
+      end
+      role = MiqUserRole.find_by_name("EvmRole-operator")
+      @group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Archived Group")
+      login_as FactoryGirl.create(:user, :userid => 'archived__wilma', :miq_groups => [@group], :settings => {})
```

Comment:
Think you want `login_as FactoryGirl.create(:user_with_group, :role => "operator")` (or maybe it is `:role => "EvmRole-operator"`)